torch/nn/_functions/thnn/activation.py

```diff
@@ -8,6 +8,11 @@
 
 
 class PReLU(Function):
+
+    @staticmethod
+    def primspec(g, input, weight):
+        return g.appendNode(g.create("PRelu", [input, weight]))
```

Comment:
Note that PReLU can work in two modes in PyTorch (weights can be shared between all channels), but it might not be true in Caffe